rpcs3/rpcs3qt/game_list_frame.cpp

```diff
@@ -354,10 +372,10 @@ void game_list_frame::Refresh(bool fromDrive)
 			GameInfo game;
 			game.root         = entry.name;
 			game.serial       = psf::get_string(psf, "TITLE_ID", "");
-			game.name         = psf::get_string(psf, "TITLE", "unknown");
-			game.app_ver      = psf::get_string(psf, "APP_VER", "unknown");
-			game.category     = psf::get_string(psf, "CATEGORY", "unknown");
-			game.fw           = psf::get_string(psf, "PS3_SYSTEM_VER", "unknown");
+			game.name         = psf::get_string(psf, "TITLE", sstr(category::unknown));
```

Comment:
since we do this in many other places as well, we will fix this seperately in another commit if necessary